include/context.h

```diff
@@ -124,4 +122,4 @@ void r_context_register_progress_callback(progress_callback progress_cb);
 
 RaucContext *r_context_conf(void);
 const RaucContext *r_context(void);
-void r_context_clean(void);
\ No newline at end of file
+void r_context_clean(void);
```

Comment:
unrelated change.

Reply:
Appears to be a line at end-of-file change done by my editor, will remove it.